_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+---
+title: "Social Media Use in 2021"
+description: |
+  % of adults using each social media.
+categories: "2022" 
+author: Álvaro Sanz 
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(fig.showtext=TRUE)
+
+```
+
+## About the graph
+
+Brooke Auxier and Monica Anderson published an article in the Pew Research Center in which they analyzed Social Media Use in 2021 for the Northamerican context, through data collected by the same organization and depicted by them in several graphs. One of them, the objective of this project, visually presents the evolution of use of eleven Social Networks since 2012 (although only 5 of them have available data for the whole time series).
+
+```{r, out.width="90%", out.height="90%", fig.align='center'}
+
+knitr::include_graphics("Data/Original plot.jpg")
+
+```
```

Comment:
Use this instead, please,

```markdown
![<caption>](Data/Original plot.jpg){.external width="90%"}
```

where `<caption>` describes the plot and links the source.